src/nashpy/linalg/tableau.py

```diff
@@ -0,0 +1,492 @@
+"""A class for the tableaus used in the Lemke Howson algorithms"""
+import warnings
+import numpy as np
+import numpy.typing as npt
+from typing import Set, List, Iterable, Optional
+
+
+class TableauBuilder(object):
```

Comment:
Given the use case, could this be a function instead of a class?

Reply:
I agree the builder feels a bit strange so happy to change here. I went into that direction to avoid a single method with too many parameters. Still if you wanted to change it would you want:

1. a single createRowTableau/createColumnTableau method with parameters for payoffs and algorithm
2. that lemke howson classs more calls individual methods to first create tableau matrix, and then possibly roll it and finally generate tableau class?

Reply:
Updated, think this is a readability improvement as well